docs/services/auth-machine/_index.md

```diff
@@ -1,16 +1,42 @@
 ---
 title: Auth-Machine
-date: 2022-03-02T00:00:00+00:00
+date: 2023-09-21T09:08:15.21901+02:00
 weight: 20
 geekdocRepo: https://github.com/owncloud/ocis
-geekdocEditPath: edit/master/docs/services/auth-machine
-geekdocFilePath: _index.md
+geekdocEditPath: edit/master/services/auth-machine
+geekdocFilePath: README.md
 geekdocCollapseSection: true
 ---
 
+<!-- Do not edit this file, it is autogenerated. Edit the service README.md instead -->
```

Comment:
Not sure if this should be part of the PR about WebDAV?!

Reply:
fallout from docs creation scripts.

Reply:
removed.